app/customer.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+
+
+current_directory = os.path.dirname(os.path.abspath(__file__))
+relative_path = os.path.join(current_directory, "config.json")
+
+
+def customers_and_content() -> list:
+    with open(relative_path, "r") as file:
+        global content
+        content = json.load(file)
+    global customers
+    customers = content.get("customers")
+    return customers, content
+
+
+class Customer:
+    def __init__(self, content: dict) -> list:
+        self.content = content
+
+    def customer_location() -> dict:
```

Comment:
Maybe it must works with self, not with all customers

Reply:
resolve comment

Reply:
not fixed